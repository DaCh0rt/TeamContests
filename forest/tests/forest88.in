294 165
795 -1435
-1678 841
-6610 -7576
4484 7883
-2457 5002
-8217 -8384
1787 -411
-4133 -5378
4653 1041
4180 2829
5987 -9114
-3469 -1533
5145 -9801
3204 -8950
-9138 7999
9882 -8354
7290 -2067
579 -7091
7287 4558
5435 7750
-208 8026
-3475 -6227
-1051 1342
-1945 -5938
-2317 -7420
3975 -3682
-3085 -8571
-7428 129
-932 2491
-2229 -9176
-1681 -6289
-2857 3432
4758 1259
7728 1751
-5227 2572
-9022 1355
-3552 302
-1621 4530
-7730 3076
1279 5886
-748 -7200
-8044 -6035
-4779 5913
4511 -585
8941 -6257
-6091 6368
2242 7253
3052 -6906
-4785 2821
-433 2294
606 6023
-9620 -6254
7835 8767
4044 -9407
-1645 8889
3984 -7634
-1543 8018
-7435 -8972
2552 -8018
7107 9897
2795 8765
-8353 4912
2409 -5417
7987 -8107
-4431 -9833
8148 3276
4549 -7024
-8821 -115
-738 894
7493 -8689
4426 3120
-57 -8754
-4774 -4085
-2674 7002
8661 4491
-9085 7424
3131 -7971
1120 -9285
6973 -2953
1918 456
3542 -3242
6536 -1036
-4655 5017
-16 -5628
6952 -1140
160 -9464
4122 -1685
-3739 2968
1801 8222
-7045 2326
-287 7112
4212 9883
-1352 -5371
9982 -534
6329 6977
-9483 2946
-1192 -6459
6941 -7389
-7758 -6746
-3407 -7120
280 8155
-1079 -7959
-7087 6200
6139 -6498
1676 4244
-7026 8503
4204 -2170
6593 5239
8645 9649
5695 8921
3556 2171
9532 -4590
-3499 -1467
-7636 -8628
-3242 -9285
7897 4529
7368 -4293
-6570 4049
-6549 3155
8303 -6311
-8906 -4729
-2139 9176
2135 2832
-1843 -9448
7362 -9900
2643 -7019
3131 -4949
9211 3184
2009 -6970
7536 -1194
-6239 9950
-5192 3646
-4102 6256
2928 1753
3083 3372
7517 4433
7617 59
-8909 -2692
1710 4461
2039 7616
-5801 3716
-798 4572
394 -2632
1322 -8656
-3734 -8106
5476 -6226
3202 1888
6990 -907
-112 -3876
-4495 4643
6751 6763
-5713 539
-559 -4205
-7197 717
-2767 4709
9172 6079
-966 -1149
-6659 -1100
7875 8341
-9412 -8706
-5209 1576
4869 -8826
-4205 -862
3869 -8064
594 -1046
6628 -2900
-5492 9583
3818 2095
-4794 1876
369 -5319
-2184 4514
-858 9353
-5882 4819
-4897 -6447
7569 9700
3248 -1845
6001 5826
-8994 -2821
-4948 3736
7620 -352
-6607 336
301 -9425
621 6469
-9955 73
-6427 -4543
-3914 -2904
-4106 3919
8072 -5000
-557 9753
-2490 4787
-1186 -2568
-3371 -1392
-4020 -4527
1725 -4236
4262 379
6778 7675
3593 3487
8578 1435
-8243 3069
4867 -6437
1512 6072
8027 6197
-8377 -3193
8417 2283
9346 -6206
491 -4104
-9375 -1240
4292 -7491
8030 -1981
3091 -9089
4456 6455
29 -128
-506 4881
-1966 8778
9409 -6363
-9699 5511
7106 -7110
-8992 2674
66 -2517
2932 259
-4180 4264
-5408 -1594
-207 -9690
-8238 -8007
652 -9625
-1688 3574
-864 -5397
-4609 322
-2892 -9668
3537 -2675
6004 3937
-5899 -7756
-6459 8796
8564 6049
-1444 -1832
536 -9916
-7202 865
1345 -431
6206 4968
-4785 -6079
-2773 -4921
-3548 -445
7153 -2070
1603 -5744
-609 -8221
8831 9508
3992 2232
-6814 -2819
-9089 -5011
8990 -4940
-7582 1420
-5213 -1459
-2714 7686
5833 -9933
-8300 6226
-8514 4425
-1839 1970
-4574 846
-8925 -847
-8298 8476
-6951 -7881
-9560 -4797
7945 6281
-9404 -1331
-2552 -3680
-687 4263
-1638 -3905
-4308 6055
8668 -2551
-8348 8821
-828 997
-5579 -4778
-5480 -6399
-1547 3920
8351 -1515
2281 -469
-9814 -7773
9219 -1929
1717 12
-1881 -7137
8279 -81
2017 4765
4968 202
7032 -6044
-913 -152
-9119 4353
1175 8184
5215 211
5365 -6095
-7975 4916
-5269 -2091
862 -9863
-2942 -9320
-4412 -6880
