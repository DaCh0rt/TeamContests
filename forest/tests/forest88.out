74349.84 387790458.16
